Gordon360/Models/ViewModels/ProfileCustomViewModel.cs

```diff
@@ -22,7 +23,8 @@ public class ProfileCustomViewModel
                 Twitter = pro.twitter ?? "",
                 Instagram = pro.instagram ?? "",
                 LinkedIn = pro.linkedin ?? "",
-                Handshake = pro.handshake ?? ""
+                Handshake = pro.handshake ?? "",
+                Calendar = pro.calendar ?? "",
```

Comment:
```suggestion
                Calendar = pro.calendar ?? ""
```
It looks like no comma is expected after the last item.